CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to dwave-examples! Before getting
+started, we encourage you to take a look at this document for some guidelines.
+
+## What We're Looking For
+
+While we have a preference for application oriented examples, we welcome all
+examples that demonstrate use of the D-Wave system.
+
+For a list of existing examples, take a look
+[here](https://cloud.dwavesys.com/leap/examples/).
+
+## Steps for Contributing
+
+1. Download this repository for a template and implement your example on a
+   personal repository. Note: To simplify the code reviewing process, we
+   encourage you to use a development branch. This way, the code may be reviewed
+   as a pull request.
+
+2. Open an issue on this repository and link to your example. See the following
+   section for what to include in your issue.
+
+3. Once your example has been reviewed, ____?
+
+## Issue Template
```

Comment:
What does "Issue Template" mean exactly?

Reply:
When external contributors submit their example, they do so by submitting an issue. The previous contribute.md had this section for indicating what should go into the issue (and our one existing contributor used it, so I'm inclined to keep it).

"Issue Template" is confusing though...perhaps just "Issue" is fine.

Reply:
On second thought, I decided to move this section to one of the bullet points in the section above. Hopefully that’s more clear.